ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -539,6 +539,36 @@ def EuclideanNorm(*, input, axis, keep_dims=False, name="EuclideanNorm"):
 ConcatV2 = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.concat))
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    filter_shape,
+    strides,
+    padding,
+    data_format="NHWC",
+    dilations=[1, 1, 1, 1],
+    name="Conv2D",
+):
+    if data_format == "NDHWC":
+        strides = [1] + strides[1:-1] + [1]
+        dilations = [1] + dilations[1:-1] + [1]
+    elif data_format == "NCDHW":
+        strides = [1, 1] + strides[2:] + [1]
+        dilations = [1, 1] + dilations[2:] + [1]
+    filter = ivy.array(ivy.xavier_init(shape=filter_shape))
+    return ivy.conv2d(
+        input,
+        filter,
```

Comment:
Ivy's second positional argument is output, so here you are calling the filter in the wrong position. Please see the docs - https://lets-unify.ai/ivy/stateful/layers/Conv2D.html#conv2d